translations/de/common.json

```diff
@@ -1 +1,81 @@
+  "abort": "Abbrechen",
+  "save": "Sichern",
+  "close": "Schließen",
+  "continue": "Weiter",
+  "back": "Zurück",
+  "version": "Version",
+  "sponsor": "Unterstützen",
+  "privacy": "Datenschutzerklärung",
+
+  "imprint": "Impressum",
+  "imprint.information": "Informationen nach Sektion 5 TMG",
+  "imprint.legal": "RECHTLICHE HINWEISE",
+  "imprint.contact": "Kontaktinformationen",
+  "imprint.email": "E-Mail",
+
+  "vaccination": "Impfung",
+  "recovery": "Genesung",
+  "test": "Test",
+
+  "meta.description": "Überprüfen Sie digitale COVID-Zertifikate der EU mit Leichtigkeit und validieren Sie sie anhand der länderspezifischen und lokalen Vorschriften",
+
+  "information.purpose": "Was ist der Zweck von Covid Validator?",
+  "information.purpose.message": "Mit Covid Validator können Sie digitale COVID-Zertifikate der EU auf der Grundlage nationaler oder lokaler Regeln scannen und überprüfen. CovidValidator zeigt der prüfenden Person nur so wenige persönliche Informationen wie möglich an, um die Privatsphäre der Nutzenden zu schützen. Diese Web-App ist eine Alternative zu anderen bestehenden Produkten, da sie nicht nur in bestimmte App-Stores oder Ländern verfügbar ist.",
+  "information.beta": "Was bedeutet <1>BETA</1>?",
+  "information.beta.message": "Die Web-App befindet sich noch in aktiver Entwicklung und kann kleinere Fehler enthalten. Der produktive Einsatz dieser Anwendung wird noch nicht empfohlen, bitte verwenden Sie die Web-App daher mit Vorsicht. Updates werden regelmäßig erscheinen. Sie können die Entwicklung auf<1>GitHub</1>verfolgen.",
+  "information.rules": "Welche Regeln werden verwendet?",
+  "information.rules.message": "Im Moment verwendet die App die offiziellen EU-Eintrittsregeln, um das Zertifikat zu validieren. Diese Regeln werden von den Mitgliedsländern des digitalen COVID Zertifikats zur Verfügung gestellt. Es ist ein bekanntes Problem, dass diese Regeln veraltet sein können und nicht den staatlichen oder lokalen Vorschriften entsprechen. Dieses Projekt arbeitet an einem benutzerdefinierten Regelersteller, der präzisere Regeln für lokale Vorschriften ermöglicht. Sie können die Entwicklung<1>hier</1>verfolgen.",
+  "information.issue": "Was ist zu tun, wenn etwas nicht funktioniert?",
+  "information.issue.message": "Überprüfen Sie zunächst, ob Sie das richtige Land und die richtigen Regeln ausgewählt haben. Versuchen Sie dann, Ihr Gerät mit dem Internet zu verbinden, um die neusten Daten herunterzuladen. Wenn das Problem weiterhin auftritt, erstellen Sie auf<1>GitHub</1>ein Issue.",
+  "information.certificate": "Was ist das digitale COVID-Zertifikat der EU?",
+  "information.certificate.message": "Weitere Informationen dazu finden Sie auf<1>der offiziellen Website der EU</1>.",
+
+  "scan.button": "Zertifikat prüfen",
+  "scan.message": "Halten Sie den QR-Code vor Ihre Kamera",
+  "scan.error": "Kamera Fehler",
+  "scan.error.message": "Haben Sie Zugriff auf die Kamera erlaubt?",
```

Comment:
Ich bin sehr unentschlossen aber müsste das nicht eher "Haben Sie den Zugriff auf die Kamera erlaubt?" heißen?

Reply:
Yes, stimmt.

Reply:
Geländert in [`0a59c8f`](https://github.com/timokoenig/covid-validator/pull/16/commits/0a59c8f0c5857c9b062b6da61d2ceea8d65d886f).